framework/aster-frame/src/sync/rcu/monitor.rs

```diff
@@ -20,6 +18,10 @@ pub struct RcuMonitor {
 }
 
 impl RcuMonitor {
+    /// Creates a new RCU monitor.
+    ///
+    /// This function is used to initialize a singleton instance of `RcuMonitor`.
+    /// The singleton instance is globally accessible via the `RCU_MONITOR`.
```

Comment:
This is the constructor method. The method itself does not implement the singleton pattern. Nor is the implementation based on such an assumption. So do not mention "singleton" here.